### ISSUE TITLE
Function to test the model

### ISSUE THREAD

**Issue by AnthonyFangqing** [AnthonyFangqing/aiprojects-nlp-team2#7]
Something like:
def predict_sentiment(text):
    model.eval() # set on eval mode. won't keep track of gradients => faster! 
    with torch.no_grad():
        test_vector = torch.LongTensor(dataset.vectorizer.transform([text]).toarray()) #format test data

        output = model(test_vector)
        prediction = torch.sigmoid(output).item()

        if prediction > 0.5:
            print(f'{prediction:0.3}: Positive sentiment')
        else:
            print(f'{prediction:0.3}: Negative sentiment')

So we can see if the model is working at all
